index.html

```diff
@@ -279,7 +279,18 @@ <h3 class="languages-title">Skills</h3>
                 </li>
 
             </section>
-
+            <section class="contact-section">
+                <h4 class="contact">Contact me</h4>
+                <p class="contact-text">If you have an application you are interested in developing, a feature that you
+                    need built or a project that needs coding. I’d love to help with it </p>
+                <form action="https://formspree.io/f/mbjbpybo" method="post" class="contact-form">
+                    <input placeholder="name" class="name" max="30" name="name" type="text" required>
+                    <input placeholder="email" class="email" name="email" type="email" required>
+                    <textarea name="message" id="message" cols="30" rows="10" maxlength="500" class="text-box"
```

Comment:
- [ ] As required, please kindly ensure that all input fields have the `required` attribute to activate the validation on submit.